pkg/webhook/controlplane/add.go

```diff
@@ -20,22 +26,44 @@ import (
 )
 
 var (
-	logger = log.Log.WithName("openstack-controlplane-webhook")
+	logger                           = log.Log.WithName("openstack-controlplane-webhook")
+	// TODO(LucaBernstein): Clean up the gardener version check after October/2024.
+	versionConstraintGreaterEqual198 *semver.ConstraintsversionConstraintGreaterEqual198 *semver.Constraints
```

Comment:
Ups, my previous suggestion was non-sense and broke the build. I am sorry about it:
```suggestion
	versionConstraintGreaterEqual198 *semver.Constraints
```

Reply:
didn't notice either. Fixed. Thanks!